Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/SANSFitShiftScale.py

```diff
@@ -0,0 +1,279 @@
+# pylint: disable=no-init,invalid-name,too-many-arguments,too-few-public-methods
+
+from mantid.simpleapi import *
+from mantid.api import DataProcessorAlgorithm, MatrixWorkspaceProperty, PropertyMode, AnalysisDataService
+from mantid.kernel import Direction, Property, StringListValidator, UnitFactory
+import numpy as np
+
+class Mode(object):
```

Comment:
This is a copy-paste of what we have in the other algorithm. Surely this could have been done in a shared module.